composer.json

```diff
@@ -55,7 +55,6 @@
     "tedivm/jshrink": "^1.3",
     "tuupola/slim-jwt-auth": "^2.0",
     "wikimedia/composer-merge-plugin": "^1.4",
-    "fullcalendar/fullcalendar": "^3.9",
```

Comment:
also was this intentional?

Reply:
Hmm, not sure. I can see that it's still in the develop branch though so it should be fine.